src/utils/graphClient.ts

```diff
@@ -0,0 +1,42 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AuthenticationProvider, Client, ClientOptions } from "@microsoft/microsoft-graph-client";
```

Comment:
```suggestion
import { AuthenticationProvider, Client as GraphClient, ClientOptions as GraphClientOptions } from "@microsoft/microsoft-graph-client";
```

And corresponding changes below